### PR TITLE
CSS : Minor fix

### PR DESCRIPTION
**Pull request by Distrib-ps**
imageoldimagenew

imageoldimagenew

imageoldimagenew

imageoldimagenew

### CODE REVIEW

File: js/client-battle.js [hunk 1]
```diff
@@ -1512,7 +1512,7 @@
 			if (this.gameType === 'battle' && this.room.battle && !this.room.battle.rated) {
 				buf += '<button type="button" name="replacePlayer" class="button">Replace player</button> ';
 			}
-			buf += '<button type="button" name="close" class="autofocus" class="button">Cancel</button></p></form>';
+			buf += '<button type="button" name="close" class="button" class="button">Cancel</button></p></form>';
```
[the remark this answers is not in the document]
Comment (Distrib-ps) [reply js/client-battle.js:1515]: oops, thx

File: js/client-battle.js [hunk 2]
```diff
@@ -1512,7 +1512,7 @@
 			if (this.gameType === 'battle' && this.room.battle && !this.room.battle.rated) {
 				buf += '<button type="button" name="replacePlayer" class="button">Replace player</button> ';
 			}
-			buf += '<button type="button" name="close" class="autofocus" class="button">Cancel</button></p></form>';
+			buf += '<button type="button" name="close" class="button">Cancel</button></p></form>';
```
Comment (urkerab) [inline js/client-battle.js:1515]: I'm pretty sure the intent was to add the `button` class without removing the `autofocus` class. Compare https://github.com/smogon/pokemon-showdown-client/blob/f9bc940eb5b2f2d55f7ea9b5c8c82e9ffe62c3c0/js/client.js#L2981
```suggestion
			buf += '<button type="button" name="close" class="autofocus button">Cancel</button></p></form>';
```
https://github.com/smogon/pokemon-showdown-client/blob/f9bc940eb5b2f2d55f7ea9b5c8c82e9ffe62c3c0/js/client.js#L2549 looks like it still needs fixing though.

File: style/client.css [hunk 3]
```diff
@@ -247,7 +247,6 @@ i.subtle:hover {
 	position: relative;
 	padding: 3px 2px 3px 2px;
 	margin: 0 -1px 0 0;
-	top: 1px;
```
Comment (Zarel) [inline style/client.css:250]: ???

File: style/battle-log.css [hunk 4]
```diff
@@ -163,6 +163,9 @@ button:disabled {
 	text-shadow: none;
 	color: #F9F9F9;
 }
+.dark select.button {
+	background-color: #2b2c31 !important;
```
Comment (Zarel) [inline style/battle-log.css:167]: I have a guess as to what's going on here but what browser is this bad???

File: js/client-teambuilder.js [hunk 5]
```diff
@@ -1202,12 +1202,12 @@
 				}
 				for (i = 0; i < this.curSetList.length; i++) {
 					if (this.curSetList.length < this.curTeam.capacity && this.deletedSet && i === this.deletedSetLoc) {
-						buf += '<li><button name="undeleteSet"><i class="fa fa-undo"></i> Undo Delete</button></li>';
+						buf += '<li><button name="undeleteSet" class="button"><i class="fa fa-undo"></i> Undo Delete</button></li>';
```
Comment (Zarel) [inline js/client-teambuilder.js:1205]: This looks better as not a button...

File: style/client.css (hunk 3 above)
Comment (Distrib-ps) [reply style/client.css:250]: I thought the spacing was intentional

File: style/battle-log.css (hunk 4 above)
Comment (Distrib-ps) [reply style/battle-log.css:167]: I use chrome

File: js/client-teambuilder.js (hunk 5 above)
Comment (Distrib-ps) [reply js/client-teambuilder.js:1205]: Would you prefer to keep it white? 
image
image

File: style/client.css (hunk 3 above)
Comment (Zarel) [reply style/client.css:250]: Sorry, I'm still confused. What is this changing and why? Do you have before/after screenshots?

Comment (Distrib-ps) [reply style/client.css:250]: Yeah, 
image
It's the little gap at the top

File: js/client-teambuilder.js (hunk 5 above)
Comment (Zarel) [reply js/client-teambuilder.js:1205]: It shouldn't be white. I must be mixing this up with something else.

Comment (Zarel) [reply js/client-teambuilder.js:1205]: Yeah, I got this mixed up with undeleting teams, which already has styling.

File: style/battle-log.css (hunk 4 above)
Comment (Zarel) [reply style/battle-log.css:167]: Hah, it turns out that Chrome for Windows has this problem, but Chrome for Mac doesn't.

Anyway, interesting solution, but it turns out styling `option` works, and doesn't require your hack that makes select buttons uglier.

**Pull request closed without merging**